tests/resources/helper_functions.py

```diff
@@ -282,4 +285,4 @@ def get_non_existent_nft_template(
         if address == ZERO_ADDRESS:
             return x
 
-    return -1
\ No newline at end of file
+    return -1
```

Comment:
In the helper function can you replace "x" with other name to be more specific. Other than that, @calina-c are you satisfied by using `range()` in the [for](https://github.com/oceanprotocol/ocean.py/pull/551/files#diff-8d3fd885d643390850dc0ddada99085063dfc0195d3f87c9f0fb4c163d144a1aR283) loop?